FxAClient/Frontend/SignIn/FxASignIn.js

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Transport postMessage events from the fxa-content-server to the embedding
+ * webview.
+ */
+
+"use strict";
+
+// We use "load" as a temporary proxy for the forth-coming "ready to paint"
+// event from the fxa-content-server, tracked by
+// https://github.com/mozilla/fxa-content-server/issues/2066.  Since
+// fxa-content-server XHR loads may not have completed at "load" time, we push
+// the event back 1000ms.
+function handleLoadEvent(evt) {
+    window.removeEventListener("load", handleLoadEvent);
+    window.setTimeout(function() {
+        webkit.messageHandlers.accountsCommandHandler.postMessage({ type: "load", detail: {command: "load", data: {}}});
+    }, 1000);
```

Comment:
Oh yeah, this wasn't pushed to dev.lcip when I was testing.  And dev.lcip is 503-ing right now.  I may land with or without this temporary hack; not clear yet.